src/App.js

```diff
@@ -18,6 +18,8 @@ import { ratingFilter } from "./utils/productFilters/rating";
 import { priceRangeFilter } from "./utils/productFilters/priceRangeFilter";
 import { fastDeliveryFilter } from "./utils/productFilters/fastDeliveryFilter";
 import { outOfStockFilter } from "./utils/productFilters/outOfStockFilter";
+import Profile from "./components/Profile/Profile";
```

Comment:
add an index.js file in components folder  then import all components into this file and then export these components as named components.
For eg:
import Profile from './Profile/Profile'
import Login from './Login/Login'
export {Profile, Login}
This way when you import your Profile component in you App.js file it will look like this:
import {Profile} from './components'

Reply:
ok will change